### PR TITLE
Initiatives top

### PR DESCRIPTION
**Pull request by keyualiceli**
initial set up of the top of initiatives page, as well fixing some font sizes for the bottom

### CODE REVIEW

**Review by kerry918 — APPROVED** [review]
Looks pretty good for me. A next step to improve might be to set a constant height for all tabs (it looks a little bit distracting when the number of events differs)

File: src/components/initiatives/Initiatives.js [hunk 1]
```diff
@@ -1,97 +1,13 @@
...
-                    <Box item p={'5rem'}>
-                        <h2 className={classes.subheader}>Tech+ 101</h2>
-                        <p className={classes.text}>Tech+ 101's main goals are helping to ease the transition of first-years this Fall 
-                            along with setting a good environment and culture for the rest of their program at UW. 
-                            Tech+ 101 will be running events, panels, and activities that help to further diversify 
-                            and build this community starting with First Years! <br/>
-                            Tech+ 101 has recently held a Q&A Panel Kick-off event and would like to host workshops, 
-                            more panels, and other events to help first-years as they needed.</p>
-                            <br />
-                        <h3><a href="#"><strong>See what we are doing!</strong></a></h3>
```
Comment (jingyi2318) [inline src/components/initiatives/Initiatives.js:60]: For both "See what we are doing!" and "Learn more" links, I was wondering if these would look better as maybe a button or something? The links right now don't feel as cohesive with the rest of the site compared to everything else. Also open to other ideas/implementing this in a later iteration, let me know what you think!

**Review by jingyi2318 — COMMENTED** [review]
I second what @kerry918  said regarding the height of the events being a bit distracting, and I also left a comment. Otherwise, this is looking really good!

File: src/components/initiatives/Initiatives.js (hunk 1 above)
Comment (eyhhuang) [reply src/components/initiatives/Initiatives.js:60]: Yeah i agree! Or even just styling the links a little more would be nice :)

**Comment by eyhhuang** [comment]
I noticed when I resize the page, a bit of the past events tab gets cut off and there's no way for me to scroll. So we should probably create an issue to handle page resizing (esp for mobile) 
image

**Pull request merged**